components/home/Challenge.tsx

```diff
@@ -105,11 +111,12 @@ export function Challenge({
                         darkColor="white"
                         type="small"
                       >
-                        {"on " + challengeDate.toLocaleDateString()}
+                        {"on " + challengeDate.toUTCString()}
```

Comment:
Same as before, it is less clear than the previous function .toLocaleDateString()

Reply:
same reply